base/terminfo_data.jl

```diff
@@ -0,0 +1,540 @@
+    TermCapability(:kf63,     :key_f63,                   "F63 function key"),
+    TermCapability(:el1,      :clr_bol,                   "Clear to beginning of line"),
+    TermCapability(:mgc,      :clear_margins,             "clear right and left soft margins"),
+    TermCapability(:smgl,     :set_left_margin,           "set left soft margin at current column. (ML is not in BSD termcap)."),
+    TermCapability(:smgr,     :set_right_margin,          "set right soft margin at current column"),
+    TermCapability(:fln,      :label_format,              "label format"),
+    TermCapability(:sclk,     :set_clock,                 "set clock, #1 hrs #2 mins #3 secs"),
+    TermCapability(:dclk,     :display_clock,             "display clock"),
+    TermCapability(:rmclk,    :remove_clock,              "remove clock"),
+    TermCapability(:cwin,     :create_window,             "define a window #1 from #2, #3 to #4,                          #5"),
+    TermCapability(:wingo,    :goto_window,               "go to window #1"),
+    TermCapability(:hup,      :hangup,                    "hang-up phone"),
+    TermCapability(:dial,     :dial_phone,                "dial number #1"),
+    TermCapability(:qdial,    :quick_dial,                "dial number #1 without checking"),
+    TermCapability(:tone,     :tone,                      "select touch tone dialing"),
+    TermCapability(:pulse,    :pulse,                     "select pulse dialing"),
+    TermCapability(:hook,     :flash_hook,                "flash switch hook"),
+    TermCapability(:pause,    :fixed_pause,               "pause for 2-3 seconds"),
+    TermCapability(:wait,     :wait_tone,                 "wait for dial-tone"),
+    TermCapability(:u0,       :user0,                     "User string #0"),
+    TermCapability(:u1,       :user1,                     "User string #1"),
+    TermCapability(:u2,       :user2,                     "User string #2"),
+    TermCapability(:u3,       :user3,                     "User string #3"),
+    TermCapability(:u4,       :user4,                     "User string #4"),
+    TermCapability(:u5,       :user5,                     "User string #5"),
+    TermCapability(:u6,       :user6,                     "User string #6"),
+    TermCapability(:u7,       :user7,                     "User string #7"),
+    TermCapability(:u8,       :user8,                     "User string #8"),
+    TermCapability(:u9,       :user9,                     "User string #9"),
+    TermCapability(:op,       :orig_pair,                 "Set default pair to its original value"),
+    TermCapability(:oc,       :orig_colors,               "Set all color pairs to the original ones"),
+    TermCapability(:initc,    :initialize_color,          "initialize color #1 to (#2, #3,                                #4)"),
+    TermCapability(:initp,    :initialize_pair,           "Initialize color pair #1 to fg=(#2, #3,                                #4),             bg=(#5,#6,#7)"),
```

Comment:
Is this wonky formatting because the source material is badly formatted?

Reply:
Ah, I thought I fixed those. I've done another pass over the file, and I think the excess spaces should now all be gone :crossed_fingers: